stores/raftstore.go

```diff
@@ -93,6 +116,45 @@ func (s *RaftStore) DeleteClient(clientID string) error {
 // RaftSubStore methods
 ////////////////////////////////////////////////////////////////////////////
 
+// CreateSub implements the SubStore interface
+func (ss *RaftSubStore) CreateSub(sub *spb.SubState) error {
+	gss := &ss.genericSubStore
+
+	gss.Lock()
+	defer gss.Unlock()
```

Comment:
Does perf matter here?

Reply:
Not that many subs can be created per second, so not really. Plus, although as you know I am in favor of using `defer` sparingly, when you start to have multiple returns in the function, it starts to make sense.

Reply:
Agree